pkg/kubectl/cmd/proxy.go

```diff
@@ -83,14 +83,33 @@ func NewCmdProxy(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringP("address", "", "127.0.0.1", "The IP address on which to serve on.")
 	cmd.Flags().Bool("disable-filter", false, "If true, disable request filtering in the proxy. This is dangerous, and can leave you vulnerable to XSRF attacks, when used with an accessible port.")
 	cmd.Flags().StringP("unix-socket", "u", "", "Unix socket on which to run the proxy.")
+	cmd.Flags().StringP("curl", "", "", "Raw URI to request from the proxy server.")
```

Comment:
OK, so:

* making request after running proxy server - done
* random port by default - not done - for now you can assign random port by `--port=0`, I have no idea whether it's fine for you to override the `8081` default port with `0` in case of using `--curl` option; please let me know what are your thoughts